pages/community/community-contribute-to-precice.md

```diff
@@ -15,7 +15,7 @@ Do you enjoy improving whatever you can? Did you find a bug in preCICE or one of
 ## Contributing documentation
 
 You (yes, you) have already found something that can be improved in this website:
-maybe an unclear part, maybe a broken link, maybe even a small typo.
+maybe an unclear part, maybe a broken link, maybe even a small tyypo.
```

Comment:
Not sure, if this joke might not backfire? :smirk: What, if somebody contributes a fix for this (intentional?) typo? Shouldn't some kind of CI pipeline ideally catch this?

Reply:
Sure, I can remove it... :cry: